.travis.yml

```diff
@@ -6,15 +6,14 @@ cache:
     - $HOME/.m2
 script:
   - mvn clean install
-  - mvn clojure:run -Dclojure.script=src/main/clojure/doc/main.clj
 branches:
   only:
   - master
 deploy:
   provider: pages
   skip-cleanup: true
   github-token: $GITHUB_TOKEN
-  local-dir: target/doc
+  local-dir: klojure-docs/target/doc
```

Comment:
Changing this to point to the docs that get generated as part of the new maven artifact.